src/Raven.Client/Documents/Session/Operations/GetRevisionOperation.cs

```diff
@@ -42,6 +42,7 @@ public GetRevisionOperation(InMemoryDocumentSessionOperations session, IEnumerab
 
         public GetRevisionsCommand CreateRequest()
         {
+            _session.IncrementRequestCount();
```

Comment:
I wonder if we don't want to have `IncrementRequestCount` in a more common place like in the `RequestExectuor.SendRequestToServer`, it is very easy to forget this increment.
For instance, does getting attachments increase the number of requests?

Reply:
cc @ppekrol  thoughts?

Reply:
What about thread-safety if we change the implementation of the incrementation to `RequestExectuor.Increment..`

Reply:
`RequestExectuor` isn't scoped to a session, it is global, so you cannot do that there.
We can't modify the API as well.

Reply:
you do have the `SessionInfo`

Reply:
That is a bigger change. Out of scope for this task. We can move it as a part of 5.3 effort.

Reply:
ok :)